api/test/docker/apisix_config2.yaml

```diff
@@ -93,4 +93,5 @@ plugin_attr:
 
   server-info:
     report_interval: 10
-    report_ttl: 3600
\ No newline at end of file
+    report_ttl: 3600
+
```

Comment:
we can remove `apisix.ssl. ssl_cert` and `apisix.ssl. ssl_cert_key`

Reply:
and we can remove `plugins` too

Reply:
same issue with file `api/test/docker/apisix_config.yaml`

Reply:
`plugins` can't be removed, because we have e2e test cases for skywalking which is not enable by default.

Reply:
I think it's not the business of this PR. Maybe we need another PR to fix it?